stellar/deploy-contract.js

```diff
@@ -116,7 +116,7 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
             const gatewayAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gateway?.address), { type: 'address' });
             const gasServiceAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gas_service?.address), { type: 'address' });
             const itsHubAddress = nativeToScVal(config.axelar?.contracts?.InterchainTokenService?.address, { type: 'string' });
-            const chainName = nativeToScVal('stellar', { type: 'string' });
+            const chainName = nativeToScVal(options.chainName, { type: 'string' });
```

Comment:
```suggestion
            const chainName = nativeToScVal(chain.axelarId, { type: 'string' });
```
Use the `axelarId`, that's the official chain identifier